src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+package com.manardenza.console;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.ReservationController;
+import com.manardenza.controller.UserController;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import com.manardenza.utils.Injector;
+import com.sun.javafx.geom.Edge;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static com.manardenza.console.VisualUserMenu.inputDataFromUser;
+import static com.manardenza.console.VisualUserMenu.outputDataInConsole;
+import static com.manardenza.console.VisualUserMenu.outputListInConsole;
+import static com.manardenza.console.VisualUserMenu.outputSplitLine;
+import static com.manardenza.console.VisualUserMenu.readerMenuFromConsole;
+import static com.manardenza.console.VisualUserMenu.showUserMenu;
+import static jdk.nashorn.internal.runtime.JSType.isNumber;
+import static org.mockito.asm.tree.InsnList.check;
+
+public class ContentsUserMenu {
+
+    private HotelController hotelController;
+    private ReservationController reservationController;
+    private UserController userController;
+    private List listData;
+
+    public ContentsUserMenu(HotelController hotelController,
+                            ReservationController reservationController, UserController userController) {
+        this.hotelController = hotelController;
+        this.reservationController = reservationController;
+        this.userController = userController;
+    }
+
+    public void mainMenu() throws IOException {
+        outputSplitLine();
+        outputDataInConsole("\t\tWelcome to manardenza.com");
+        outputSplitLine();
+        outputDataInConsole("\n\tAuthorization");
+        showUserMenu("Main menu");
+        int actionMain = readerMenuFromConsole("Choose action: ");
+        switch (actionMain) {
+            case (1):
+                registrationUserMenu();
+                break;
+            case (2):
+                outputDataInConsole("\tBuy!");
+                System.exit(0);
+                break;
+            default:
+                outputDataInConsole("Incorrect menu item");
+                mainMenu();
+                break;
+        }
+    }
+
+    public void serviceMenu() throws IOException {
+        boolean flagMenu = true;
+        while (flagMenu) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS135)